docs/content/new-tutorial/intro-to-assets.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: "Introduction to the Dagster Tutorial and Assets | Dagster Docs"
+description: "Learn about the objectives for the Dagster tutorial and about assets"
+noindex: True
+---
+
+# Introduction to Dagster
+
+Dagster is an orchestration framework to help data engineers deliver high-quality data. In Dagster, the objects made by data engineers are called assets.
+
+In this tutorial, you will write a data pipeline that performs ETL (Extract, Transform, Load). You will:
+
+- **Extract** data from a website
+- **Transform** it into a data structure for a chart
+- And **load** it into a visualization for reporting
+
+While writing this pipeline, you will learn about Dagster's core concepts, such as assets, schedules, and I/O Managers.
+
+## Introduction to software-defined assets
```

Comment:
I think this is an area where a lot of readers will grasp what a software-defined asset is a lot more quickly if they can see the code that defines one. I think I personally would feel a little impatient if I had to read a bunch before seeing some code.  I wonder if we should jump in to the code part of the tutorial here?

Reply:
E.g. "Dagster encourages you to define assets in software - i.e. to write Python code that that describes the assets you want to exist and how they're computed. Here's a first asset:".

Reply:
My guess, and very possible I'm wrong here - is that the combo of code + Dagit visualization is enough to get across what's going on here. I have experimented with a few different SDA visualizations in the past and haven't found any of them amazingly satisfying.